migrations/1_deploy_contracts.js

```diff
@@ -58,7 +58,7 @@ module.exports = async function (deployer, network) {
       packetAddress
     );
     const mockTransferAddress = await deployContract("MockTransfer", ibcAddress);
-    const transferAddress = await deployContract("ICS20TransferERC20", ibcAddress);
+    const transferAddress = await deployContract("ICS20TransferERC20Allowlist", ibcAddress);
```

Comment:
Why should this be changed?

Reply:
because I demand this contract requires the ability of setting third party ERC20 contract, in the case of users sending ERC20 token from Axon to CKB

Reply:
Let's not change this or it'll break existing ibc tests.